apps/prairielearn/src/middlewares/studentAssessmentRedirect.ts

```diff
@@ -0,0 +1,30 @@
+import { Router } from 'express';
+
+import { loadSqlEquiv, queryOptionalRow } from '@prairielearn/postgres';
+
+import { IdSchema } from '../lib/db-types.js';
+
+const router = Router();
+const sql = loadSqlEquiv(import.meta.url);
+
+router.all('/', async function (req, res, next) {
```

Comment:
Nit: rather than being a router, it'd be more conventional to just export a function. See e.g. `selectAndAuthzAssessment`, `selectAssessments`, etc.

Reply:
Updated to a function rather than a router in [451d697](https://github.com/PrairieLearn/PrairieLearn/pull/10889/commits/451d6970eef032732ab367e2aec3b1c55c3af28e).